src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/test/WattTimeDataSourceTests.cs

```diff
@@ -1,321 +1,322 @@
-﻿using CarbonAware.Exceptions;
```

Comment:
### ⚠️ ALERT ⚠️ 
line-ending changes are making this diff go bananas 🍌 .   I am fixing it